docs/features/localnet.md

```diff
@@ -16,6 +18,10 @@ If you are on Windows then you will need WSL 2 installed first, for which you ca
 
 Alternatively, the Windows 10/11 Pro+ supported [Hyper-V backend](https://docs.docker.com/desktop/install/windows-install/) for Docker can be used instead of the WSL 2 backend.
 
+### Podman support
+
+If you prefer to use [Podman](https://podman.io/) as your container engine, make sure to install and configure `podman` first. Then you can set the default container engine to use by executing `algokit localnet config podman`. See [Container-based LocalNet](#container-based-localnet) for more details.
```

Comment:
As discussed, let's move this to `algokit config localnet`

Will need to be updated everywhere.